.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# Configuration for the GitHub feature to automatically request reviews from the code owners when a pull request
+# changes any owned files.
+#
+# References:
+#   https://github.com/blog/2392-introducing-code-owners
+#   https://help.github.com/articles/about-codeowners
+
+# +--------------------+
+# + Default Code Owner +
+# +--------------------+
```

Comment:
Add me for the backend folder and apiclient in the frontend.
For the rest of the frontend, please split the ownership with @MarcoJul